setup.py

```diff
@@ -29,9 +29,9 @@
         "keras==2.9.0",
         "tensorflow-gpu==2.9.3",
         "wandb~=0.12.0",
-        "umap~=0.1.1",
+        "umap-learn~=0.1.1",
         "seaborn~=0.12.2",
-        "chemprop @ git+https://github.com/soulios/chemprop.git@1d73523e49aa28a90b74edc04aaf45d7e124e338",
+        "chemprop~=1.7.1",
```

Comment:
well, we would lose the training tracking(which isnt that important) and one change in the interpret function to save it in a csv instead of just logging it in the terminal.